Users/Docker_Image.rst

```diff
@@ -63,7 +64,8 @@ on your code with a ``.travis.yml``, like this:
 
     language: generic
     services: docker
-    script: docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
+    script: docker run --volume=$(pwd):/app --workdir=/app coala/base \
+            coala --ci
```

Comment:
I'd prefer it was one line, so there is less change someone copy&pastes incorrectly.
IMO change `--volume=..` to `-v=...` to shorten it.